go.mod

```diff
@@ -3,13 +3,60 @@ module github.com/iamkirkbater/osd-readiness-spike
 go 1.15
 
 require (
-	github.com/openshift/api v0.0.0-20200521101457-60c476765272
-	github.com/openshift/client-go v0.0.0-00010101000000-000000000000
-	k8s.io/apimachinery v0.19.4
-	k8s.io/client-go v0.18.3
+	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
+	github.com/openshift/client-go v0.0.0-20200326155132-2a6cd50aedd0
+	k8s.io/apimachinery v0.19.2
+	k8s.io/client-go v12.0.0+incompatible
+)
+
+require (
+	github.com/go-delve/delve v1.5.1 // indirect
+	github.com/mattn/go-colorable v0.1.8 // indirect
+	github.com/mattn/go-runewidth v0.0.10 // indirect
+	github.com/openshift/osde2e v0.0.0-20210127203025-7b09b7d9f79d
```

Comment:
This will need to be updated with a new hash once https://github.com/openshift/osde2e/pull/681 lands.

Reply:
Done.